core/sms_server.py

```
@@ -0,0 +1,1422 @@
+    global pool
+    if pool is not None:
+        try:
+            await pool.close()
+            logger.info("Database pool closed")
+        except Exception as e:
+            logger.error(f"Error closing database pool: {e}")
+    
+    logger.info("SMS Bridge shutdown complete")
+
+@app.post("/sms/receive")
+async def receive_sms(request: Request, background_tasks: BackgroundTasks):
```

Ruff (0.14.6)

Unused function argument: `background_tasks`

(ARG001)